src/controls/map/Map.tsx

```diff
@@ -35,29 +33,34 @@ export class Map extends React.Component<IMapProps, IMapState> {
   /**
    * componentWillMount lifecycle hook
    */
-  public componentWillMount(): void {
-    this.setState({
-      coordinates: this.props.coordinates
-    });
-  }
+  // 20200614 - legacy react lifecycle event; use constructor instead
```

Comment:
We're still using React v 15 in this library, so it's not legacy for this version.
Anyway, we can remove this one.